src/Symfony/Component/DomCrawler/UriExpanderTrait.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DomCrawler;
+
+/**
+ * Any HTML element that can link to an URI.
```

Comment:
It could be worth be more explicit and telling ppl when it's worth using this trait.

Reply:
Actually, I think this should be a new class instead of a trait.
If we merge as a trait, all private methods are going to be covered by the BC policy.
We don't want that IMHO.